examples/seismic/abc_methods/01_introduction.ipynb

```diff
@@ -0,0 +1,1187 @@
+{
```

Comment:
<p>RickerSource is a class not a command. This is constructed.</p><p> </p><ul><li><em>time_range</em> is the <em>time grid</em> that we are using in our simulations;</li></ul><p>Time range is the range of the timesteps not a time grid.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
About RickerSource OK! But the time_range have more informations than timesteps, like the number of timesteps, intial time, final time and teh size of timestep, so this is the time grid (grid of all time informations). Is not correct consider in this way?

Reply:
Well, from this perspective yes. But we do not want users to confuse that with the typical Devito Grid construction. So I suggest that we remove the term time grid.

Reply:
OK, I change this term in the text! This probleme is solved!